### ISSUE TITLE
Add LICENSE.txt or LICENSE or LICENSE.md

### ISSUE THREAD

**Issue by bmhm** [java-native/jssc#22]
The code from scream3r is LGPL.
I suggest we add the according license to each file and to the root of the github project.

**Comment by tresf** [comment]
> I suggest we add the according license to each file and to the root of the github project.

FYI, the license (specifically LGPL 3.0) is already mentioned in each file.

Here's an example I've used in LGPL projects:

```java
/**
 * @author Johnny Doe
 *
 * Copyright (C) 2018 Johnny Doe
 * LGPL 2.1 This is free software.  This software and source code are released under
 * the "LGPL 3.0 License".  A copy of this license should be distributed with
 * this software. https://opensource.org/licenses/lgpl-3.0.html
```

... and then `LICENSE.TXT` with the actual license body.  We may also choose to simply leave the existing headers and just upload a `LICENSE.TXT`.

**Comment by bmhm** [comment]
Sure, leave the headers and add the `LICENSE` file. As soon as I see it there, I'll add it to the `pom.xml` as mentioned in #23.

**Comment by tresf** [comment]
Added to `master`. ![image](https://user-images.githubusercontent.com/6345473/55816893-52738080-5ac1-11e9-9998-ebe9b953c7f4.png)

**Comment by tresf** [comment]
@bmhm I'll leave the `pom.xml` stuff to you and then we can close this out as completed, right?

**Comment by tresf** [comment]
> Add LICENSE to pom.xml. Fixes #22. 2d31761

`<rant>`... minor but I'm not that excited about explicitly attaching LGPL 3.0 in source but then hard-linking to whatever gnu.org has at "lgpl.html", which is subject to change.

Fortunately... since the license specifically mentions...

> or * (at your option) any later version.

... then we're in the clear.  Note, LGPL 3.0 has been the same for about 12 years, I'd expect this link to have been the same for most of that time.  For these reasons, `lgpl.html` probably won't change anytime soon -- but once it does -- we would need to link to the "old version" section where 3.0 gets stashed at that time.  Alternately, we may choose to bump to 4.0, etc by modifying the source files.

`</rant>`

@bmhm thanks for finishing this up.  Closing as the license is clear in GitHub, source and Maven.
